packages/nimble-components/src/spinner/styles.ts

```diff
@@ -0,0 +1,216 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    Black15,
+    Black91,
+    White
+} from '@ni/nimble-tokens/dist/styledictionary/js/tokens';
+import { Theme } from '../theme-provider/types';
+import { hexToRgbaCssColor } from '../utilities/style/colors';
+import { themeBehavior } from '../utilities/style/theme';
+
+export const styles = css`
+    ${display('inline-block')}
+
+    :host {
+        width: 32px;
+        height: 32px;
+    }
+
+    :host([size='small']) {
+        width: 16px;
+        height: 16px;
+    }
+
+    :host([size='medium-large']) {
```

Comment:
If we are going to use the t-shirt metaphor, let's stick to those sizes. XS, S, M, L, XL, XXL, and so on.

Reply:
Updated to use more standard size names XS-XXL:
`x-small` (16px), `small` (32px), `medium` (48px), `large` (64px), `x-large` (96px), `xx-large` (128px)